src/supplemental/nanolib/conf_ver2.c

```diff
@@ -1127,10 +1128,17 @@ static void
 conf_exchange_parse_ver2(conf *config, cJSON *jso)
 {
 	cJSON *node_array = hocon_get_obj("exchange_client", jso);
+	cJSON *jso_parquet = cJSON_GetObjectItem(jso, "parquet");
 	cJSON *node_item  = NULL;
 
+	conf_exchange *conf_exchange = &config->exchange;
+	if(conf_exchange->exchange_url == NULL) {
+		conf_exchange->exchange_url = nng_zalloc(64);
```

Comment:
Do not need allocate memory, hocon_read_str will use strdup.

Reply:
oh shit